README.md

```diff
@@ -185,7 +194,7 @@ To run the web server in development mode use the following command. This must b
 flask --app sandman_web run --debug --host 0.0.0.0
 ```
 
-Then in your web browser enter the following URL: YOUR_SANDMAN_IP_ADDRESS:5000/reports. Currently you must provide the report suffix, because index redirection is not working at the moment.
+Then in your web browser enter the following URL: YOUR_SANDMAN_IP_ADDRESS:5000/reports. Currently you must provide the report suffix, because index redirection is not working at the moment. You can stop the web server with CTRL + C.
```

Comment:
Now that you have fixed the index redirection, let's change this to just point at the index without the caveat. I would also specify that you are pressing control C in the TERMINAL.

Reply:
I agree with these two suggestions. The changes have been committed.